### PR TITLE
Replace withTranslation HOC with hooks and test against actual i18n'd values

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: src/containers/CollapsibleSection.js [hunk 1]
```diff
@@ -1,9 +1,3 @@
-import { compose } from 'redux';
-import { withTranslation } from 'react-i18next';
 import { CollapsibleSection } from '../components/CollapsibleSection';
 
-const enhance = compose(
-  withTranslation(),
-);
-
-export default enhance(CollapsibleSection);
+export default CollapsibleSection;
```
Comment (marlo-longley) [inline src/containers/CollapsibleSection.js:3]: Is this container doing anything now? Is it needed?

File: src/containers/LocalePicker.js [hunk 2]
```diff
@@ -1,9 +1,3 @@
-import { compose } from 'redux';
-import { withTranslation } from 'react-i18next';
 import { LocalePicker } from '../components/LocalePicker';
 
-const enhance = compose(
-  withTranslation(),
-);
-
-export default enhance(LocalePicker);
+export default LocalePicker;
```
Comment (marlo-longley) [inline src/containers/LocalePicker.js:3]: Same question whether this is now doing anything

File: src/containers/OpenSeadragonViewer.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { compose } from 'redux';
 import { connect } from 'react-redux';
-import { withTranslation } from 'react-i18next';
 import flatten from 'lodash/flatten';
+import { withTranslation } from 'react-i18next';
```
Comment (marlo-longley) [inline src/containers/OpenSeadragonViewer.js:4]: I think this one didn't get deleted (here and below in `enhance`)

Comment (cbeer) [reply src/containers/OpenSeadragonViewer.js:4]: OpenSeadragonViewer is still a class (until #4014), so we're stuck with the HOC.

**Comment by cbeer** [comment]
Yes, I think those are the next steps, and I'm hopeful this'll merge nicely into the vite branch (and if it doesn't, it should just be removing lines of code...)

**Comment by marlo-longley** [comment]
Needs rebase but looks good to me.

**Review by marlo-longley — DISMISSED** [review]

**Review by marlo-longley — APPROVED** [review]

**Pull request merged**